server/lib/packets/topic.go

```diff
@@ -0,0 +1,43 @@
+package packets
+
+import (
+	"log"
+
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/models"
+)
+
+func ProcessTopicCreate(from Source, msgid uint32, req *proto.TopicCreateRequest, t Transport) {
+	tp, err := models.NewTopic(models.DBM(), from.Account(), req)
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: models
* [gotype] tp declared but not used